souschef.py

```diff
@@ -25,19 +28,146 @@
 
 """ Additional Constants """
 ###########################################################
+BASE_URL = 'https://www.betterworlded.org/try'
+
+# Read csv file
+
+# csv file name
+filename = "bwe_overall_database.csv"
 
+# initializing titles and rows list
+fields = []
+rows = []
 
 """ Main Scraping Method """
 ###########################################################
 def scrape_source(writer):
 	""" scrape_source: Scrapes channel page and writes to a DataWriter
         Args: writer (DataWriter): class that writes data to folder/spreadsheet structure
         Returns: None
+
+        Better World Ed is organized with the following hierarchy:
+            Grade Level (Folder) ***There may not be an assigned grade level*** (there should be 24 unassigned grade level rows)
+            |   Math Topic (Folder)  ***May come as a hyperlink***
+			|   |   Specific Objective (Folder) ***May come as a hyperlink***
+			|   |   |   Written Story (File)
+			|   |   |   Video (File)
+			|   |   |   Lesson Plan (File)
+
+
+		Additional notes:
+
+		1)
+		There are some cases where we would want to remove forward slashes (/), from the titles of files or folders.
+		This is because when trying to name a folder, the PATH variable and Datawriter will create a new folder, for example:
+		the 120th record, row 4, has the hyperlink, "I Am Shantanu // Chai & Community". This will cause the Datawriter to
+		add a folder, "I Am Shantanu", with a file inside named " Chai & Community", with a preceding whitespace, instead
+		of a file "I am Shantanu // Chai & Community".
+
+		2)
```

Comment:
Can remove this note, but maybe also add a note that you've updated the downloader function as `add_file` doesn't account for downloading from google